pkg/restclient/volume.go

```diff
@@ -3,7 +3,9 @@ package restclient
 import (
 	"fmt"
 	"net/http"
+	"net/url"
 
+	"github.com/gluster/glusterd2/glusterd2/volume"
```

Comment:
Restclient package should not import any of the core packages except `api` packages.